release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft.Win32.SafeHandles
+
+``` diff
+ namespace Microsoft.Win32.SafeHandles {
+     public sealed class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvalid {
+-        public override bool IsInvalid { get; }
```

Comment:
Is this a breaking change?

Reply:
I think its a mistake. this is the change that removed it https://github.com/dotnet/runtime/commit/4fed6ab8818fd33298f8ec1cfbe6ffe2387d4637
@ericstj do we need to add this back ?

Reply:
It's not breaking, it's an override.  Code compiled against the old definition would still work at runtime and just call the base.